earthmover/earthmover.py

```diff
@@ -88,6 +91,10 @@ def __init__(self,
 
         # Set current working directory to the location of the config file.
         os.chdir(os.path.dirname(self.config_file))
+
+        # convert state_configs to YamlMapping so we can inject CLI overrides
+        self.state_configs = YamlMapping().update(self.state_configs)
+        self.state_configs = self.inject_cli_overrides(self.state_configs, "config.")
```

Comment:
I want to inject the CLI overrides into `state_configs` (so things like `show_stacktrace` can be overridden), but they need to be a `YamlMapping` for that to work... so I create that here.